data/about-us.js

```diff
@@ -0,0 +1,12 @@
+const getAboutUsData = () => ({
```

Comment:
Is there a reason why this is a function and not just an object or even a json?

Reply:
Yes that's because sometimes I pass values here. For example on the homepage I pass the number of stars which was fetched before, in order to construct a sentence with this variable. If I'm not using it yet on this page, do you prefer that I remove the function ?

Reply:
Good for me :) Maybe it can be a nice good first issue for contributors to handle